pkg/edit/vars.go

```diff
@@ -31,6 +34,22 @@ func addVar(fm *eval.Frame, name string, val any) error {
 	return nil
 }
 
+func delVar(fm *eval.Frame, name string) error {
+	if !isUnqualified(name) {
+		return errs.BadValue{
+			What:  "name argument to edit:del-var",
+			Valid: "unqualified variable name", Actual: name}
+	}
+
+	variable := fm.Evaler.Global().IndexString(name)
+	if variable == nil {
+		return errors.New("no variable $" + name)
+	}
+
```

Comment:
There's a race condition here when the global namespace is swapped out after the check but before the `.DeleteGlobal` call.

For dynamic deletion calls, I feel it's best to just ignore keys that don't exist. This is how `dissoc` and `del E:FOO` behave today.